www/tutorial/README.md

```diff
@@ -0,0 +1,4 @@
+## InSpec Demo
```

Comment:
Please include the steps to generate content in the readme aka running the rake task. Do we need to adapt the rake task, since the directory changed.